components/org.wso2.carbon.identity.scim2.common/src/test/java/org/wso2/carbon/identity/scim2/common/impl/SCIMUserManagerTest.java

```diff
@@ -111,6 +114,8 @@
 @PowerMockIgnore("java.sql.*")
 public class SCIMUserManagerTest extends PowerMockTestCase {
 
+    final static public String TENANT_DOMAIN = "carbon.super";
```

Comment:
```suggestion
    public static final String TENANT_DOMAIN = "carbon.super";
```

Reply:
Since this is used only inside the class, can't we make this private?

Reply:
We can get rid of this constant and use `org.wso2.carbon.utils.multitenancy.MultitenantConstants.SUPER_TENANT_DOMAIN_NAME` instead.